src/libraries/Microsoft.Extensions.Logging.Console/tests/Microsoft.Extensions.Logging.Console.Tests/ConsoleLoggerExtensionsTests.cs

```diff
@@ -419,13 +419,24 @@ public void AddConsole_NullFormatterName_UsingDefaultFormat_IgnoreFormatterOptio
 #pragma warning disable CS0618
             Assert.True(logger.Options.DisableColors);
 #pragma warning restore CS0618
-            var formatter = Assert.IsType<SimpleConsoleFormatter>(logger.Formatter);
-            
-            Assert.False(formatter.FormatterOptions.SingleLine);                    // ignored
-            Assert.Equal("HH:mm:ss ", formatter.FormatterOptions.TimestampFormat);  // ignore FormatterOptions, using deprecated one
-            Assert.False(formatter.FormatterOptions.UseUtcTimestamp);               // not set anywhere, defaulted to false
-            Assert.False(formatter.FormatterOptions.IncludeScopes);                 // setup using lambda wins over config
-            Assert.Equal(LoggerColorBehavior.Disabled, formatter.FormatterOptions.ColorBehavior);                  // setup using lambda
+            if (ContainerUtils.IsContainer)
```

Comment:
>IsContainer [](start = 31, length = 11)

do we have any test running inside a real container?

Reply:
I updated the tests to exercise the container path expecting Json formatter when running in a docker container.

Reply:
@safern does the CI exercise the path when we are running in a docker container?

Reply:
Yes, there should be various tests that run inside a container, i.e Alpine, Fedora, Windows Nano and CentOS.